### DISCUSSION TITLE
Equipmet and respawn

### DISCUSSION THREAD

**Discussion by ColoradoM3** [Vdauphin/HeartsAndMinds#1606] [Question and answer] [answered]
Hello, I would have 2 questions. 1. How could I modify the equipment with which you appear in vanilla? 2. How can I make the respawn position be where I have placed the slot or the respawn module? Since when you disconnect and go to connect the slot it stays in its last position.

**Comment by Vdauphin** [comment, marked as answer]
Hello,

1.  https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/define_mod.sqf#L51
2. H&M is a persistent mission so the vehicle you use will stay with you even after a restart, just use it to RTB

Cheers
